examples/data-generator/package.json

```diff
@@ -1,7 +1,9 @@
 {
-    "name": "data-generator",
+    "name": "data-generator-retail",
```

Comment:
the name `data-generator` is already taken

Reply:
`ra-data-generator` ?

Reply:
well, we may publish data generators for blogs, social network, etc in the future. Also, it's in react-admin for now, but honestly nothing forbids that it lives on its own. It's just more convenient to keep it here for now.